include/ui/cocoa.h

```diff
@@ -87,6 +90,8 @@ typedef struct {
  */
 - (BOOL) isMouseGrabbed;
 - (BOOL) isAbsoluteEnabled;
+- (BOOL) isFullGrabEnabled;
+- (BOOL) isSwapOptionCommandEnabled;
```

Comment:
These methods are purely internal ones so remove this from this common header file.

Reply:
isFullGrabEnabled is actually called from a static function (the callback that the global tap calls), so I think it needs to be here? I removed the other, will push.

Reply:
A method is callable from a static function if the implementation appears before the caller, just as a function is in plain old C.